modules/storage-selinux-file-context-relabeling.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+
+:_content-type: CONCEPT
+[id="selinux-file-context-relabeling_{context}"]
+= SELinux File Context Relabeling
+
+When a pod is created and requires a volume to be mounted, the CRI-O container runtime is instructed by the node's kubelet process upon pod creation to relabel the entire volume with proper SELinux contexts.
+
+There are three methods for skipping the SELinux relabeling for a volume:
```

Comment:
Should we add some background here? Do we need mention it is applicative because/when a pod may not start or takes an excessive amount of time to start when it's pv contains a very high number of files.
cc @gnufied @gcharot

Reply:
Maybe we could add link https://access.redhat.com/solutions/6221251 in this part.